### PR TITLE
docs: define rules of the challenge

### PR DESCRIPTION
**Pull request by scarmuega**

### CODE REVIEW

File: rules.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+### Pilot
+
+- the pilot is a Cardano native token from a particular policy id that is locked in an address managed by the participant. We call this class of token `PilotToken`.
+- the goal of this token is to ensure that changes to the `ShipState` are triggered by the rightful owner of the ship.
+- the pilot token needs to be present in any of the inputs of a transaction that progresses the state of the ship.
+
+### Fuel Pellet
+
+- a fuel pellet is freely available source of fuel tokens that are located at a fixed position in the grid.
+- a fuel pellet is represented by a UTxO that contains any amount of `FuelToken` and is locked at a specific script address. We call this class of UTxO `PelletState`
```
Comment (manugunther) [inline rules.md:65]: Why tokens for fuel? For the first version it's simpler to have the fuel represented by an integer in the datum, in the same way the ship fuel is represented.

File: rules.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+        pos_x: Int, // x coordinate of the current position
+        pos_y: Int, // y coordinate of the current position
+        pos_ts: PosixTime, // timestamp of last movement
+        fuel: Int // total amount of available fuel
+    }
+}
+```
+
+> ::question::
+> should we handle fuel as part of the datum or rely on a native token?
```
Comment (manugunther) [inline rules.md:51]: I propose to represent the fuel as an integer in the datum. Both for ship status and fuel pellet status.

File: rules.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+## Components
+
+### Grid
+
+- the grid is a virtual space in 2 dimensions (`x`, `y`).
+- there's no on-chain representation of the grid
+- coordinates on each dimension are discrete (integer values)
+- the coordinates `(0, 0)` are considered the center of the grid
+
+### Ship
```
Comment (manugunther) [inline rules.md:26]: We need a way to represent the max fuel capacity of a ship. The ship should be able to charge fuel up to a predefined amount.

Comment (scarmuega) [reply rules.md:26]: yes, defined as `MAX_SHIP_FUEL` in the gathering fuel action.

**Review by manugunther — APPROVED** [review]

**Pull request merged**